### PR TITLE
Adds scroll button to all the pages

### LINKED ISSUE #997: Add scroll to top button on all the pages
Currently only tracks page has the button to scroll to top, it would be great if we have it on all the pages to provide a unified and a better user experience.

@aayusharora @Princu7 we should add the button to all the pages , what do you think?

### PR DESCRIPTION
**Pull request by geekyd**
This fixes #997 adds scroll to top button on all the pages.
@aayusharora Please review.

### CODE REVIEW

**Comment by Princu7** [comment]
@geekyd Nice. Please also provide a link to the live gh-pages of the event site. Thanks!!

File: src/backend/templates/partials/scroll.hbs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<a href="#" id="down-button" data-scroll-direction="0" style="display: flex;">
```
Comment (aayusharora) [inline src/backend/templates/partials/scroll.hbs]: @geekyd Try to keep this in .scss

File: src/backend/generator.js [hunk 2]
```diff
@@ -16,9 +16,11 @@ const ftpDeployer = require('./ftpdeploy');
 
 const navbar = handlebars.compile(fs.readFileSync(__dirname + '/templates/partials/navbar.hbs').toString('utf-8'));
 const footer = handlebars.compile(fs.readFileSync(__dirname + '/templates/partials/footer.hbs').toString('utf-8'));
+const scroll = handlebars.compile(fs.readFileSync(__dirname + '/templates/partials/scroll.hbs').toString('utf-8'));
 
 handlebars.registerPartial('navbar', navbar);
 handlebars.registerPartial('footer', footer);
+handlebars.registerPartial('scroll', scroll);
```
Comment (aayusharora) [inline src/backend/generator.js]: @geekyd Awesome !! :+1:

**Comment by geekyd** [comment]
@aayusharora I have made the requested changes please review it.

@Princu7 please check the link for live demo
https://open-event-web-appp.herokuapp.com/

:)

**Comment by Princu7** [comment]
@geekyd Sorry for the delay in reviewing. Could you please squash your commits, please? Thanks!!

**Comment by geekyd** [comment]
@Princu7 @aayusharora Squashed the commits!
:)

**Comment by Princu7** [comment]
Looks Nice. Thanks!!

**Pull request merged**